cluster-autoscaler/cloudprovider/gce/mig_info_provider_test.go

```diff
@@ -28,25 +29,26 @@ import (
 
 var (
 	errFetchMig             = errors.New("fetch migs error")
+	errFetchMigInstances    = errors.New("fetch mig instances error")
 	errFetchMigTargetSize   = errors.New("fetch mig target size error")
 	errFetchMigBaseName     = errors.New("fetch mig basename error")
 	errFetchMigTemplateName = errors.New("fetch mig template name error")
 	errFetchMigTemplate     = errors.New("fetch mig template error")
 
-	gceRef = GceRef{
-		Project: "project",
-		Zone:    "us-test1",
-		Name:    "mig",
-	}
 	mig = &gceMig{
-		gceRef: gceRef,
+		gceRef: GceRef{
+			Project: "project",
+			Zone:    "us-test1",
+			Name:    "mig",
+		},
 	}
 )
 
 type mockAutoscalingGceClient struct {
```

Comment:
Not related to this PR, but I'd love to see that extracted to a separate file, so we could start using it in other tests. For 90% of tests this is so much better than the Http mock we use.
Though with this refactor maybe a MockMigInfoProvider would be even better?

Reply:
I think we have some more refactors for that to be possible - still there are some client functions which are called by manager directly. But a bit more and I think we could stop mocking both client and cache, and instead mock only various providers.